assets/src/edit-story/app/canvas/canvasProvider.js

```diff
@@ -43,9 +43,9 @@ function CanvasProvider({ children }) {
   const [pageAttachmentContainer, setPageAttachmentContainer] = useState(null);
   const [displayLinkGuidelines, setDisplayLinkGuidelines] = useState(false);
 
-  const { pageSize, setPageSize } = useLayout(({ state, actions }) => ({
-    pageSize: state.canvasPageSize,
-    setPageSize: actions.setCanvasPageSize,
+  const pageSize = useLayout(({ state: { pageWidth, pageHeight } }) => ({
```

Comment:
Should `useLayout` be used everywhere for getting the `pageSize` perhaps and `pageSize` removed from returning from `CanvasProvider` at all? `useCanvas` is currently still used in one place only (in `pagemenu/index.js`). Thoughts?

Reply:
Hm, that's a good point. Didn't even check how many instances there were of this. Fixed in 6bd6863a3 with follow-up in f48462ee9 and 68bca1504.